run

```diff
@@ -1,6 +1,27 @@
 #!/bin/bash
 set -euo pipefail
 
+log() {
+    msg="$1"
+
+    logline="[$(pwd) $0] $msg\n"
```

Comment:
why is `$(pwd)` important for every log line?

Reply:
There are several files logging, and even multiple files named `run`, so the `$(pwd)` helps with debugging / context. I can pull it into a var like `LOG_PREFIX` so that it isn't run every time though.